.github/workflows/stale-issues.yml

```
@@ -0,0 +1,42 @@
+name: Check stale issues
+
+on:
+  workflow_dispatch: # FIXME: remove
+  schedule:
+    - cron: '0 12 * * *'
+
+env:
+  PERIOD: 14 # in days
+  WAIT_FOR_INFO: "waiting for more input"
+  DRY_RUN: true
+
+jobs:
+  check-issue-status:
+    name: Retrieve all open issues that are inactive
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get inactive open issues
```

This would be more readable if steps were a bit more separated.
```suggestion
      - uses: actions/checkout@v3

      - name: Get inactive open issues
```